### ISSUE TITLE
A bug occurred when training the network to calculate loss

### ISSUE THREAD

**Issue by 1287217526** [savinay95n/PatchRefineNet#3]
In src/deepglobe/refine-models/train.py,
the code for calculating the loss is written as 
rst_loss = loss_rst(y1, pred)
reg_loss = loss_orig(y2, pred)
From the definition of loss, the positions of y and pred should be swapped, otherwise it will lead to loss calculation errors or even nan situations
